Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,165 @@
+Parser:
```

Comment:
Obviously, all comments above to start.

Reply:
I don't think you replicated, at least for now, the fixes in the ASIM parser here.

Reply:
yes, I will do this when all is resolved on ASim

Reply:
replicated